servers/Azure.Mcp.Server/README.md

```diff
@@ -62,41 +62,48 @@ From within Visual Studio 2022 install [GitHub Copilot for Azure (VS 2022)](http
 1. Install the [GitHub Copilot](https://plugins.jetbrains.com/plugin/17718-github-copilot) plugin.
 1. Install the [Azure Toolkit for Intellij](https://plugins.jetbrains.com/plugin/8053-azure-toolkit-for-intellij) plugin.
 
-### Additional IDEs
-
-For IDEs not listed above, manually setup Azure MCP Server within the IDE.
+### Manual IDE Setup
```

Comment:
Should we say `Manual MCP Client Setup` since not everything listed below is an actual IDE?

Reply:
I can do that